time    time_recognized
string  final_result
string  partial_result 
bool    isSpeech_recognized